man/timesyncd.conf.xml

```diff
@@ -103,6 +103,17 @@
         Defaults to 30 seconds and must not be smaller than 1 second.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>SaveIntervalSec=</varname></term>
+        <listitem><para>Interval to periodically save current time. This is especially useful for offline
+        systems with no RTC battery, as it will guarantee that the time increases monotonically between
+        reboots.</para>
+
+        <para>Takes a time span value. The default unit is seconds, but other units may be specified, see
```

Comment:
"time interval" seems a little more precise to me

Reply:
Done.